target/linux/brcm47xx/image/Makefile

```diff
@@ -536,146 +587,167 @@ endif
 
 ifeq ($(SUBTARGET),mips74k)
 define Device/asus-rt-ac53u
-  DEVICE_TITLE := Asus RT-AC53U
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-AC53U
   DEVICE_PACKAGES := $(USB2_PACKAGES)
   $(Device/asus)
   PRODUCTID := RT-AC53U
 endef
 TARGET_DEVICES += asus-rt-ac53u
 
 define Device/asus-rt-ac66u
-  DEVICE_TITLE := Asus RT-AC66U
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-AC66U
   DEVICE_PACKAGES := kmod-b43 $(USB2_PACKAGES)
   $(Device/asus)
   PRODUCTID := RT-AC66U
 endef
 #  TARGET_DEVICES += asus-rt-ac66u
 
 define Device/asus-rt-n10
-  DEVICE_TITLE := Asus RT-N10
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-N10
   DEVICE_PACKAGES := kmod-b43
   $(Device/asus)
   PRODUCTID := "RT-N10      "
 endef
 TARGET_DEVICES += asus-rt-n10
 
 define Device/asus-rt-n10p
-  DEVICE_TITLE := Asus RT-N10P
+  DEVICE_VENDOR := Asus
+  DEVICE_MODEL := RT-N10P
   DEVICE_PACKAGES := kmod-b43
```

Comment:
Add v1 (?) here.

Reply:
The first version does not seem to have an explicit revision.

Reply:
done